views/tests.ejs

```diff
@@ -11,8 +11,8 @@ See the LICENSE file for copyright details
 <%- include('extensioncheck', {sendToServer: true}) %>
 <p id="status">Running tests...</p>
 <form id="export" action="/export" method="post">
-  <button id="export-download" type="submit" name="download" value="download" disabled><span class="mdi mdi-download"></span> Export for download</button>
-  <button id="export-github" type="submit" name="github" value="github" disabled><span class="mdi mdi-github"></span> Export to GitHub</button>
+  <button id="export-download" type="submit" name="download" value="download" disabled><span class="mdi mdi-download"></span> Download results file</button>
+  <button id="export-github" type="submit" name="github" value="github" disabled><span class="mdi mdi-github"></span> Send results as a GitHub PR</button>
```

Comment:
Maybe adding a bit more verbosity to exactly where the results are going would be helpful?

```suggestion
  <button id="export-github" type="submit" name="github" value="github" disabled><span class="mdi mdi-github"></span> Send results to mdn-bcd-results GitHub repo</button>
```